share/spice/frequency_allocation/frequency_allocation.js

```diff
@@ -0,0 +1,62 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_frequency_allocation = function(api_result) {
+
+        if (!api_result || api_result.status == 'Fail' || api_result.Errors) {
+            return Spice.failed('frequency_allocation');
+        }
+        
+        var script = $('[src*="/js/spice/frequency_allocation/"]')[0],
+            source = $(script).attr("src"),
+            query = source.match(/frequency_allocation\/([^\/]+)/)[1],
+            targetFrequency = decodeURIComponent(query);
+        
+        
+        var infoboxData;
+        if (targetFrequency >= 3000 && targetFrequency < 30000){
```

Comment:
Should the upper constraint here be `3700` since that is the upper constraint of the API?